templates/definition/meter/varta.yaml

```diff
@@ -23,6 +23,8 @@ params:
     default: 502
   - name: capacity
     advanced: true
```

Comment:
Konsequenterweise müsste das `maxdischargepower` heissen, gerne mit Beschreibung in die defaults.yaml einfügen.